projects/storefrontlib/cms-components/navigation/breadcrumb/breadcrumb.component.ts

```diff
@@ -21,8 +29,17 @@ import { PageTitleComponent } from '../page-header/page-title.component';
   templateUrl: './breadcrumb.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class BreadcrumbComponent extends PageTitleComponent implements OnInit {
+export class BreadcrumbComponent
+  extends PageTitleComponent
+  implements OnInit, OnDestroy
+{
   crumbs$: Observable<any[]>;
+  protected subscription = new Subscription();
+
+  private router = inject(Router);
```

Comment:
I think `router` should at most be `protected`. The `featureConfigService` is only set to `private` because we plan to delete it.